File: 1012.py
# 1012 - Area
pi = 3.14159
a = float(input())
b = float(input())
c = float(input())

i = (a*c)/2
j = pi*pow(c, 2)
k = ((a+b)/2)*c
l = b*b
m = a*b

print("TRIANGULO: %0.3f\n" % i)
print("CIRCULO: %0.3f\n" % j)
print("TRAPEZIO: %0.3f\n" % k)
print("QUADRADO: %0.3f\n" % l)
print("RETANGULO: %0.3f\n" % m)

File: 1012.cpp
// 1012 - Area
#include <iostream>
#include <math.h>
using namespace std;

int main(){
    float a, b, c, pi = 3.14159;

    cin >> a >> b >> c;

    float i = (a*c)/2;
    float j = pi*pow(c, 2);
    float k = ((a+b)/2)*c;
    float l = b*b;
    float m = a*b;

    printf("TRIANGULO: %0.3f\n", i);
    printf("CIRCULO: %0.3f\n", j);
    printf("TRAPEZIO: %0.3f\n", k);
    printf("QUADRADO: %0.3f\n", l);
    printf("RETANGULO: %0.3f\n", m);

	return 0;
}
